apps/desktop-wallet/vite.config.ts

```diff
@@ -36,10 +37,9 @@ export default defineConfig({
       define: {
         global: 'globalThis'
       }
-    },
-    include: ['@alephium/sdk'] // To allow for using npm link https://vitejs.dev/guide/dep-pre-bundling.html#monorepos-and-linked-dependencies
+    }
   },
-  plugins: [react(), viteTsconfigPaths(), svgrPlugin()],
+  plugins: [react(), viteTsconfigPaths(), svgrPlugin(), nodePolyfills()],
```

Comment:
I actually notice that there's a console error after unlocking a wallet, maybe it's related to this... To be investigated.